### ISSUE TITLE
Event Metadata window in Main view

### ISSUE THREAD

**Issue by AndrewMohawk** [redcanaryco/mac-monitor#11]
**At a high level -- can you summarize your request?**
If I have a lot of events that look similar it would be nice to quickly be able to use the arrow keys to navigate down the list while looking at the event metadata to see something like if there are different parent processes for these. Even if its just a non usable text "summary" view of the event facts underneath the mini-chart

**What is the current alternative solution?**
Open each event in event metadata and manually compare

**Comment by Brandon7CC** [comment]
This is really cool the more I think about it... but I have some pretty strict design guidelines of my own that I want to keep to. I'll need to think about how we add some additional UI elements / views to show these bits of info. Since this would be a larger design shift this isn't something I think I can add to my backlog right now, but keeping it noted that the community would like some additional UI elements / views in the app!

**Comment by AndrewMohawk** [comment]
Yeah I can definitely see that. I'd even be okay with the event metadata window updating as I move through the events in the table!
